sourcegraph/cody
# Fix telemetry event cody.auth.login:firstEver firing too often in agent

Closes CODY-2346

## Test plan

1. Start JetBrains extension with this branch using `CODY_DIR` param
2. See in debugger that `setHasAuthenticatedBefore` is called instead of `handleFirstEverAuthentication`
3. Check in looker to check that `cody.auth.login:firstEver` event was not fired
